Makefile

```diff
@@ -52,6 +52,7 @@ release:
   	  -DCMAKE_INSTALL_PREFIX="${PREFIX}"       \
   	  -DMSCORE_INSTALL_SUFFIX="${SUFFIX}"      \
   	  -DMUSESCORE_LABEL="${LABEL}"             \
+          -DUSE_SYSTEM_FREETYPE="${USE_SYSTEM_FREETYPE}" \
```

Comment:
a) too many spaces
b) I guess you want that in the debug section too
c) what about macOS and Windows?